pyanaconda/modules/storage/platform.py

```diff
@@ -142,14 +145,24 @@ def stage1_descriptions(self):
             "mdarray": _(RAID_DESCRIPTION)
         }
 
+    @property
+    def stage1_constraints(self):
+        """The platform-specific constraints for the stage1 device."""
+        constraints = {
+            PLATFORM_DEVICE_TYPES: ["disk"]
+        }
+        return dict(super().stage1_constraints, **constraints)
```

Comment:
Wouldn't it be better to just change the original dict?
```
constraints = super().stage1_constraints
constraints[PLATFORM_DEVICE_TYPES] = ["disk"]
```

Reply:
I have tried that and it is less readable when there are more then one changes.